tests/boardtest.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+git submodule update --init autohat
+cd autohat
+export application_name=`echo $testNode | tr '[:upper:]' '[:lower:]'`
```

Comment:
Looking at the script it seems this variable does not need to be exported. Does it?